packages/clay-link/package.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "@clayui/link",
+	"version": "3.0.0",
+	"description": "ClayLink component",
+	"license": "BSD-3-Clause",
+	"repository": "https://github.com/liferay/clay/tree/master/packages/clay-link",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "lib/ClayLink.js",
+	"esnext:main": "src/ClayLink.js",
+	"jsnext:main": "src/ClayLink.js",
+	"files": ["lib", "src"],
+	"scripts": {
+		"build": "babel src --root-mode upward --out-dir lib --extensions .ts,.tsx",
+		"build:types": "tsc --project ./tsconfig.declarations.json",
+		"prepublishOnly": "npm run build && npm run build:types",
+		"start": "webpack-dev-server --mode development",
+		"test": "jest --config ../../jest.config.js"
+	},
+	"keywords": ["clay", "react"],
+	"dependencies": {
+		"classnames": "^2.2.6"
+	},
```

Comment:
I updated the generator in this commit, https://github.com/liferay/clay/pull/1781/commits/7013a80bf26af2ae429cc4d807528e642e98cfe3, we need to make sure to include clay-css as a devDependency since we use it for the demo App.js

Reply:
We have `clay-css` as` devDependencies` in the root folder, would not this resolve our issue?

Reply:
Oh I had never noticed that. we might be fine then.